changelog.d/20250909_101651_30907815+rjmello_user_runtime_platform_sc_44808.rst

```diff
@@ -0,0 +1,27 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- We've deprecated the ``user_runtime.python_version`` reserved template variable in favor
+  of more granular variables to facilitate leveraging Python installation information in
+  the configuration template:
+
+  - ``user_runtime.python.version``: Python version string (e.g., ``"3.13.7"``)
+  - ``user_runtime.python.version_tuple``: Python version as a tuple (e.g., ``(3, 13, 7)``)
+  - ``user_runtime.python.version_info``: Python version info from ``tuple(sys.version_info)``
+    (e.g., ``(3, 13, 7, "final", 0)``)
+  - ``user_runtime.python.compiler``: Python implementation (e.g., ``"CPython"``)
+  - ``user_runtime.python.implementation``: String identifying the compiler used for compiling
+    Python (e.g., ``"Clang 14.0.6"``)
+
+  The ``user_runtime.python_version`` variable provides the complete ``sys.version`` string,
+  which contains useful information but is hard to utilize in the template.
+
+- Added reserved template variables containing general platform information from the user's
+  installation:
```

Comment:
Consider inline linking to the documentation (for version 3.14.0, which won't exist until this is released!).  One or both of:
- `.../3.14.0/endpoints/templates.html#reserved-variables` (perhaps something like ``:ref:`... <reserved_variables>` ``?)
- `../3.14.0/reference/client.html#globus_compute_sdk.sdk.batch.UserRuntime`

Reply:
Merging the PR, but will likely follow up on this.